qiskit/pulse/library/parametric_pulses.py

```diff
@@ -70,6 +71,14 @@ def __init__(
                              amplitude is constrained to 1.
         """
         super().__init__(duration=duration, name=name, limit_amplitude=limit_amplitude)
+
```

Comment:
We should put something in the docstring about this having been supersceded by `SymbolicPulse` and that it will be deprecated and eventually removed in the future because of that